terraform/outputs.tf

```diff
@@ -33,22 +28,6 @@ output "cluster_endpoint" {
   value       = "cluster"
 }
 
-output "metrics_endpoint" {
-  description = "Name of the endpoint to provide the prometheus_scrape interface."
-  value       = "metrics-endpoint"
-}
-
-output "grafana_dashboard_endpoint" {
-  description = "Name of the endpoint to provide the grafana_dashboard interface."
-  value       = "grafana-dashboard"
-}
-
-output "logging_endpoint" {
-  description = "Name of the endpoint to provide the loki_push_api relation interface."
-  value       = "logging"
-}
```

Comment:
**question:** These were there, but are obviously replaced by the `cos-agent` endpoint. 
Why were these there in the first place? For K8s?

Reply:
If I am being honest I am not totally sure. The file in the first place was written by a different team

Reply:
If I had to guess I would say they were there earlier since the grafana-subordinate now integrates to those end points?